### DISCUSSION TITLE
Laminator suggestions

### DISCUSSION THREAD

**Discussion by ThSGM** [MiczFlor/RPi-Jukebox-RFID#1556] [Ideas]
According to this post: https://splittscheid.de/phoniebox-bauanleitung-toniebox-alternative/#ftoc-karten-laminieren

You can use 60mmx90mm lamination films in order to laminate the RFID cards, which you will need to cut by about 3mm (I guess with scissors? Or a specialised cutter?)

What I am concerned about is which laminators to choose and if they support the required thickness. For example, a cheap UK laminator like [this one ](https://www.amazon.co.uk/Rexel-Style-Office-laminator-2104511/dp/B07RFWBR59/)supports 125 microns. 

Can people please share their experiences with using laminators for RFID cards? Are there easier and better ways?

**Comment by steveleader** [comment]
I just bought this one without much investigation and it (luckily?) worked flawlessly with the rfid cards
https://www.amazon.de/gp/aw/d/B004UV1I0Q

**Reply by ThSGM** [reply]
Annoyingly, that one doesn't seem available in the UK, but from what I can see, there are loads of laminators and they're probably all the same. 

When you laminated the RFID cards, did you: 

- Have to cut the cards?
- What size and thickness pouch did you use?

**Reply by steveleader** [reply]
I guess you are right, i assume this is a standard product which is avialable under different names.

To your questions:
I used these pouches https://www.amazon.de/gp/product/B003DM1Q4Q/ They are 60x90cm and 125µm thickness.
I had to cut the cards just as described by splittscheid, but that was no problem at all.
During the lamination process I had to feed them through the laminator twice, lenghtway and across, in order to seal all sides.

All in all a very easy process with a surprisingly durable result :).

**Reply by ThSGM** [reply]
Thanks! And another basic question: did you use a special cutter to snip RFID cards? Or just scissors? 

Oh, by the way, on Amazon UK, they seem to also sell 65mmx95mm pouches in addition to 60x90. I wonder if that is enough to laminate without cutting. How much did you have to snip off?

**Reply by steveleader** [reply]
yes, I just used common scissors - wasn't hard to cut them. I you are unsure if you might cut the antenna or the chip, you can hold the card against a bright light source, then you can see through. But the cut is so far to the side, that there is no danger of cutting some elements.

65x95 sounds promising. I had to cut approx. 5mm off the witdh - so that could be a perfect fit. And another 5mm in height should also do no harm. I guess 65x95 is no mistake even if some cutting could be still required

Frankly, I haven't thought about that until now, but I can find them at amazon.de as well - hmm. I think I have to order them as well :) Thank you for that hint! And if you go for them - let me know how they worked out :)
